wikipedia-dark.user.css

```diff
@@ -6546,6 +6546,49 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   .list-switcher-element a[role="button"] {
     color: var(--base-color) !important;
   }
+  *[style*="background:#c0cfe4"],
+  *[style*="background:#9999DF"] {
+    background-color: var(--blue-4) !important;
+  }
+  *[style*="background:#a0ade3"],
+  *[style*="background:#ccccff"] {
+    background-color: var(--blue-3) !important;
+  }
+  *[style*="background:#e2e4c0"],
+  *[style*="background:#d0cfa4"] {
+    background-color: var(--yellow-1) !important;
```

Comment:
`#e2e4c0` as yellow-1 is way overkill youde be better sending these to brown colors.

Reserve very bright colors on bg for warning boxes borders etc

Reply:
I think you're right, brown-2 looks good

Reply:
brown-3 looks too dark I think.. Also I can't find a place where there is #d0cfa4 anymore so maybe this is not so important

Reply:
How do you make github print the color with its hex?

Reply:
You just type the hex code surrounded by backticks.
```
 `#3300cc`
 ```
 And you get `#3300cc`.